### PR TITLE
Issue 8 - "how to" contribute draft

### PR DESCRIPTION
**Pull request by sjay72**
Hi Andrew, 
I may have confused the terms "framework" and "template" unintentionally. Appreciate any clarifications you'd like to make. And any other changes or additions you'd like - goes without saying. :-)
Susan

### CODE REVIEW

**Comment by adahms** [comment]
@sjay72 - Thank you for the pull request!

Looks good, and I'll pass this one over to @lbopf for review.

**Comment by lbopf** [comment]
Hey @sjay72 - It's looking good :) I added a bunch of comments and suggestions inline as I read through. A few small nits about markup and contractions, and then some suggestions for things to add or remove. Let me know if I can provide more context on anything, or if there are any parts you think we should discuss further!

File: doc-Community_Collaboration_Guide/topics/contributing_to_the_framework.adoc [hunk 1]
```diff
@@ -1,4 +1,171 @@
...
+When using the SSH address for creating a clone, you need to add an SSH Key to GitHub.
+
+. Open a terminal window.
+. Check to see if you have a public SSH key:
++
+[options="nowrap",subs="+quotes"]
+----
+$ ls ~/.ssh/
+----
+. If you don't have a key, generate one:
```
Comment (sjay72) [inline doc-Community_Collaboration_Guide/topics/contributing_to_the_framework.adoc]: do not

**Comment by sjay72** [comment]
@adahms, I didn't intend to merge this myself w/out final review. But I did "git push -f" and see now that "force" does more than push the commits to the origin!
@lbopf, Thank you very much for the review comments. I made all the changes you requested.

**Pull request closed without merging**